public/react-chat/app.jsx

```diff
@@ -1,5 +1,5 @@
 // A placeholder image if the user does not have one
-const PLACEHOLDER = '/placeholder.png';
+const PLACEHOLDER = 'https://placeimg.com/60/60/people';
```

Comment:
Because the guide pages originally had a link to placehold.it then it go changed to `/placeholder.png` which meant that if you went through any of the guides in the docs you just got a broken image link.